app/config/errors.php

```diff
@@ -10,61 +10,64 @@
     /** General Errors */
     Exception::GENERAL_UNKNOWN => [
         'name' => Exception::GENERAL_UNKNOWN,
-        'description' => 'An unknown error has occured. Please check the logs for more information.',
+        'description' => 'An unknown error has occured. Please submit a support ticket at [TODO: Where do people get help?]() with reproduction steps for further assistance.',
         'code' => 500,
     ],
     Exception::GENERAL_MOCK => [
         'name' => Exception::GENERAL_MOCK,
-        'description' => 'General errors thrown by the mock controller used for testing.',
+        'description' => 'General errors thrown by the mock controller used for testing. If you\'re not building Appwrite from source, please raise an issue at [TODO: Where do people get help?]()',
```

Comment:
Is there a places we want to direct users who need support? Maybe an email address?

Reply:
I've never seen anyone ask about this exception. Maybe they should reach out on discord so we can have a discussion about what they're doing and why they're getting this.

Reply:
Okay, this should never occur on Cloud so I'm gonna say "raise an issue on GitHub or reach out to us on Discord."